test/e2e/kubevirt.go

```diff
@@ -313,16 +315,31 @@ var _ = Describe("Kubevirt Virtual Machines", func() {
 			}
 		}
 
-		httpServerTestPodsMultusNetworkIPs = func(netName string) map[string][]string {
+		checkPodRunningReady = func() func(Gomega, *corev1.Pod) {
+			return func(g Gomega, pod *corev1.Pod) {
+				GinkgoHelper()
```

Comment:
hmm I don't think you need `GinkgoHelper` for functions that receive a gomega instance from gomega

Reply:
@qinqon ^ ??

Reply:
ALl right, I will remove it after CI finish so we have results

Reply:
Done; removed it.